src/applications/hca/config/migrations.js

```diff
@@ -254,7 +254,7 @@ export default [
     }
 
     // temp fix until we get insurance v2 enabled
-    (formData?.providers || []).forEach(policy => {
+    for (const policy of formData?.providers || []) {
```

Comment:
Minor performance update here to allow for a clean "break" out of the loop when the first invalid item is discovered.